src/tests/ftest/vmd/fault_reintegration.yaml

```diff
@@ -0,0 +1,79 @@
+hosts:
+  test_servers: 2
+  test_clients: 1
+# If we define the server under test_servers, launch.py will convert it to the
+# actual server name passed into --test_servers. If above test_servers is hosts,
+# it'll be used as one of the servers at test startup time, so use something
+# other than hosts.
```

Comment:
Recommend removing this, since launch.py documents how servers/clients are handled